queries.sql

```diff
@@ -73,4 +73,12 @@ GROUP BY s.name;
 --Audit Runtime
 EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animals_id = 4;
 EXPLAIN ANALYZE SELECT * FROM visits where vets_id = 2;
-EXPLAIN ANALYSE SELECT * FROM owners where email = 'owner_18327@mail.com';
\ No newline at end of file
+EXPLAIN ANALYSE SELECT * FROM owners where email = 'owner_18327@mail.com';
+
+--  CREATE INDEXES 
+ CREATE INDEX ON visits (animals_id); 
+ CREATE INDEX ON visits (vet_id); 
+ CREATE INDEX ON owners (email);
```

Comment:
- [ ] As the previous reviewer mentioned, Please add indexes for all foreign keys as per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/database_from_diagram.md#:~:text=Remember%20to%20add%20the%20FK%20indexes.) and kindly remove the indexes from here to the `schema` file just below the tables you created.

Reply:
@Omar-Muhamad  thanks 
i fixed it .
![indx](https://user-images.githubusercontent.com/100702612/191285383-0889a70d-78d4-4ea9-880d-94e45774a3a5.png)